database/challenge.go

```diff
@@ -238,28 +245,16 @@ func (cr *ChallengeResponse) LoadFromBytes(b []byte) error {
 	return nil
 }
 
-// LoadFromReader loads a ChallengeResponse from the given io.Reader.
-//
-// Typically, this reader will be a request.Body.
-func (cr *ChallengeResponse) LoadFromReader(r io.Reader) error {
-	// Parse the request's body.
-	b, err := ioutil.ReadAll(r)
-	if err != nil {
-		return errors.AddContext(err, ErrInvalidChallengeResponse.Error())
-	}
-	return cr.LoadFromBytes(b)
-}
-
 // LoadString loads a PubKey from its hex-encoded string form.
 func (pk *PubKey) LoadString(s string) error {
-	bytes, err := hex.DecodeString(s)
+	bb, err := hex.DecodeString(s)
```

Comment:
not important but sometimes I have trouble trying to figure out your variable names, what does `bb` stand for?

Reply:
Hm, this is actually pretty important. Both `b` and `bb` usually stand for a `[]byte`. I don't like using `bytes` because it's a package name and once you shadow it, you can't use its functions. I usually use `b`, which is our standard approach in `skyd`.